kod/object/active/holder/nomoveon/battler/monster/towns/kocatwn/kcsmith.kod

```diff
@@ -107,7 +107,7 @@ messages:
 
    UserEntered(who = $)
    {
-      Post(poOwner,@SomeoneSaid,#what = self, #string=kcSmith_entry_welcome,
+      Send(poOwner,@SomeoneSaid,#what = self, #string=kcSmith_entry_welcome,
```

Comment:
You still need to revert all of these changed files that are not related to this PR.